app/stacks/uk-south/appeals-service/terragrunt.hcl

```diff
@@ -77,6 +78,7 @@ inputs = {
   back_office_service_bus_namespace_name          = dependency.back_office_ukw.outputs.service_bus_namespace_name
   service_bus_appeals_bo_case_topic_id            = dependency.back_office_ukw.outputs.service_bus_appeals_bo_case_topic_id
   service_bus_appeals_bo_document_topic_id        = dependency.back_office_ukw.outputs.service_bus_appeals_bo_document_topic_id
+  service_bus_odw_listed_building_topic_id        = dependency.back_office_ukw.outputs.service_bus_odw_listed_building_topic_id
```

Comment:
there appears to be a mix of ukw and uks references in here
thinking this should potentially be fixed in another PR